### PR TITLE
More debugger options and fixes

### PR DESCRIPTION
**Pull request by Zychronix**

### CODE REVIEW

File: src/main/java/legend/game/combat/SEffe.java [hunk 1]
```diff
@@ -4142,6 +4143,12 @@ public static void tickAdditionOverlaysEffect(final ScriptState<EffectManagerDat
 
                   //LAB_80107664
                   final long v1 = joypadPress_8007a398.get();
+
+                  if (Config.autoAddition()) {
```
Comment (LordMonoxide) [inline src/main/java/legend/game/combat/SEffe.java:4147]: Extra space

File: src/main/resources/legend/game/debugger/debugger.fxml [hunk 2]
```diff
@@ -1,17 +1,10 @@
...
-<?import javafx.scene.control.Menu?>
-<?import javafx.scene.control.MenuBar?>
-<?import javafx.scene.control.MenuItem?>
-<?import javafx.scene.control.Spinner?>
-<?import javafx.scene.layout.AnchorPane?>
-<?import javafx.scene.layout.HBox?>
-<?import javafx.scene.layout.VBox?>
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
```
Comment (LordMonoxide) [inline src/main/resources/legend/game/debugger/debugger.fxml:5]: Is intellij still collapsing these even with the formatter changes?

File: src/main/java/legend/core/Config.java [hunk 3]
```diff
@@ -69,6 +70,20 @@ public static boolean changeBattleRGB() {
     return readBool("battle_ui_colour_change", false);
   }
 
+  public static void toggleBattleUIColour() { properties.setProperty("battle_ui_colour_change", String.valueOf(!changeBattleRGB())); }
```
Comment (LordMonoxide) [inline src/main/java/legend/core/Config.java:73]: I would prefer to have these on different lines to minimize diffs if they change

**Review by LordMonoxide — CHANGES_REQUESTED** [review]

File: src/main/java/legend/game/combat/SEffe.java (hunk 1 above)
Comment (Zychronix) [reply src/main/java/legend/game/combat/SEffe.java:4146]: removed extra space

File: src/main/resources/legend/game/debugger/debugger.fxml (hunk 2 above)
Comment (Zychronix) [reply src/main/resources/legend/game/debugger/debugger.fxml]: IntelliJ automatical does this, maybe scenebuild one of the two. Imports added back.

File: src/main/java/legend/core/Config.java (hunk 3 above)
Comment (Zychronix) [reply src/main/java/legend/core/Config.java]: All on new lines now.

File: src/main/java/legend/game/combat/SEffe.java (hunk 1 above)
Comment (Zychronix) [reply src/main/java/legend/game/combat/SEffe.java]: Updated again

**Pull request closed without merging**